src/Map/config/icon.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\UX\Map\Icon\MapIconRenderer;
+use Symfony\UX\Map\Icon\MapIconRendererInterface;
+
+return static function (ContainerConfigurator $container): void {
+    $container->services()
+        ->set('.ux_map.ux_icons.renderer', MapIconRenderer::class)
+        ->args([
+            service('.ux_icons.icon_renderer')->ignoreOnInvalid(),
+        ])
+        ->alias(MapIconRendererInterface::class, '.ux_map.ux_icons.renderer')
+    ;
```

Comment:
Can't we move this configuration to the `services.php`, but replace `ignoreOnValid()` by `nullOnInvalid()`?